packages/react/src/components/Tag/Tag.js

```diff
@@ -38,8 +38,8 @@ const Tag = ({
   onClose,
   ...other
 }) => {
-  const tagId = id || `tag-${getInstanceId()}`;
-  const tagClasses = classNames(`${prefix}--tag`, className, {
+  const tagClass = `${prefix}--tag--${type}`;
```

Comment:
Which version is the right one from master?

Reply:
the deleted is from master. 
It was the reverted issue. The new line came from the cherry-picked commits.

Reply:
Could we update this to whatever is in `master`?